src/library/scala/collection/mutable/SortedMap.scala

```diff
@@ -68,6 +68,25 @@ object SortedMap extends SortedMapFactory.Delegate[SortedMap](TreeMap) {
 
     override def empty: WithDefault[K, V] = new WithDefault[K, V](underlying.empty, defaultValue)
 
+    @deprecated("Consider requiring an immutable Map or fall back to Map.concat.", "2.13.0")
+    override def + [V1 >: V](kv: (K, V1)): WithDefault[K, V1] =
+      (underlying + kv).withDefault(defaultValue)
+
+    @deprecated("Use ++ with an explicit collection argument instead of + with varargs", "2.13.0")
+    override def + [V1 >: V](elem1: (K, V1), elem2: (K, V1), elems: (K, V1)*): WithDefault[K, V1] =
+      (underlying + (elem1, elem2, elems: _*)).withDefault(defaultValue)
+
+    @deprecated("Consider requiring an immutable Map.", "2.13.0")
+    @`inline` override def -- (keys: IterableOnce[K]): WithDefault[K, V] = {
+      lazy val keysSet = keys.toSet
+      fromSpecificIterable(this.filterKeys(k => !keysSet.contains(k))).withDefault(defaultValue)
+    }
+
+    override def concat[V2 >: V](suffix: collection.Iterable[(K, V2)]): WithDefault[K, V2] =
+      underlying.concat(suffix).withDefault(defaultValue)
+
+    /*@`inline` final*/ override def ++ [V2 >: V](xs: collection.Iterable[(K, V2)]): WithDefault[K, V2] = concat(xs)
```

Comment:
TODO: the same ` if (transformedUnderlying.eq(underlying)) this else ...` trick as the other implementations.